ebseg/Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM julia:1.9-bookworm
```

Comment:
I don't think this file is needed. It for sure doesn't need any of the Julia parts, but also since the beset code isn't here, I would drop it. Git history will have the commits with the reps that @hollandjg will need